docs/src/faq.rst

```diff
@@ -22,7 +22,7 @@ Please follow these steps unless you know what you are doing.
 
   - ''Register Anaconda as my default Python 3.x''. Finish installation.
 
-4. Manually add Anaconda to path by following the instructions that can be found `here <https://www.computerhope.com/issues/ch000549.htm>`_.
+4. Navigate to the folder containing your Anaconda distribution. This folder contains multiple subfolders. Please add the path to the folder called `condabin` to your *PATH* environmental variable. This path should end in `Anaconda3/condabin`. You can add paths to your *PATH* by following these `instructions <https://www.computerhope.com/issues/ch000549.htm>`_.
```

Comment:
Is there no option during the installation process for this? Else add a qualifier to check first?

Reply:
Yes there is, but it sometimes interferes with conda init, so I would not advise to do that. This is how it works for sure.